testing/test_configs/valid_github_origin.yaml

```diff
@@ -0,0 +1,2 @@
+origin: https://github.com/user/project
```

Comment:
Could you also add valid configs for all supported issue trackers:
 * Without a scheme provided - e.g. `github.com/user/project`
 * With `www` provided - e.g. `www.github.com/user/project` and `https://www.github.com/user/project`

Reply:
added in defa509